autodoc/data/admin-api.lua

```diff
@@ -390,6 +391,33 @@ return {
         },
       },
     },
+    metadata = {
+      title = [[ Metadata ]],
+      description = [[ Developer-supplied data. ]],
+      ["/metadata/plugins/:name"] = {
+        GET = {
+          title = [[ A plugin's metadata. ]],
```

Comment:
the title field usually starts with an action verb, with no period:
```suggestion
          title = [[Retrieve the metatada for a plugin ]],
```